library/core/src/slice/mod.rs

```diff
@@ -163,7 +163,11 @@ impl<T> [T] {
     #[inline]
     #[must_use]
     pub const fn first(&self) -> Option<&T> {
-        if let [first, ..] = self { Some(first) } else { None }
```

Comment:
fix: it looks like you might have auto-formatted this with different settings from what rustc uses?

You can use `./x.py fmt` to fix it.

@rustbot author

Reply:
Oh sorry, that was a mistake. I have my editor set to auto-format Rust code.